src/components/typography/Body.tsx

```diff
@@ -24,7 +24,13 @@ const legacyFontName: IOFontFamily = "TitilliumSansPro";
  */
 export const Body = forwardRef<View, BodyStyleProps>(
   (
-    { weight: customWeight, color: customColor, asLink, ...props },
+    {
+      weight: customWeight,
+      color: customColor,
+      asLink,
+      accessibilityRole = "link",
```

Comment:
If we define "link" as a default value if `asLink` is false the component would have link as role anyway, so maybe we should leave it undefined and handle the undefined case only when the `asLink` is true, what do you think?

Reply:
That's not quite true, because we only pass the `accessibilityRole` prop when `asLink` is `true`. If you look at the generated snapshots, nothing really changes.

Does it make sense?

Reply:
yeah correct, I missed the ternary on IOText, that's correct!